src/__tests__/acceptance/test-helper.ts

```diff
@@ -0,0 +1,32 @@
+import {FluffyOctoEngineApplication} from '../..';
+import {
+  createRestAppClient,
+  givenHttpServerConfig,
+  Client,
+} from '@loopback/testlab';
+
+export async function setupApplication(): Promise<AppWithClient> {
+  const restConfig = givenHttpServerConfig({
+    // Customize the server configuration here.
+    // Empty values (undefined, '') will be ignored by the helper.
+    //
+    // host: process.env.HOST,
+    // port: +process.env.PORT,
+  });
```

Comment:
:robot: OpenAI

The commented out `host` and `port` configuration options should be removed since they are not used.
```suggestion
const restConfig = givenHttpServerConfig({
  // Customize the server configuration here.
  // Empty values (undefined, '') will be ignored by the helper.
});
```

<!-- This is an auto-generated comment by OpenAI -->